mon-pix/app/components/navbar-desktop-header.js

```diff
@@ -13,7 +13,7 @@ export default class NavbarDesktopHeader extends Component {
 
   get menu() {
     const menuItems = this._menuItems;
-    return this.isUserLogged ? menuItems.filterBy('permanent', true) : menuItems;
+    return (this.isUserLogged || this._isFromExternalPlatform) ? menuItems.filterBy('permanent', true) : menuItems;
```

Comment:
J'ai cherché à comprendre ce qu'était les permanent menu items `menuItems.filterBy('permanent', true)`. Je ne vois pas où cette propriété `permanent` est définie, il n'y a aucune mention dans le code.

Et ce n'est pas [un mot clef](https://api.emberjs.com/ember/release/functions/@ember%2Fobject%2Fcomputed/filterBy)

**Solution 1**
Enlever cette mention et garder la fonctionnalité
```suggestion
    return (this.isUserLogged || this._isFromExternalPlatform) ? [] : menuItems;
```

**Solution 2**
Utiliser cette proprité
``` 
      { name: this.intl.t('navigation.not-logged.sign-in'), link: 'login', class: 'navbar-menu-signin-link', permanent: false },
      { name: this.intl.t('navigation.not-logged.sign-up'), link: 'inscription', class: 'navbar-menu-signup-link', permanent: false },
```

Reply:
Je prends la solution 1, plus simple, et aussi proposée par @chrc .

Reply:
Les grands ... se rencontrent